### ISSUE TITLE
Xcode 8.3 Image: Duplicate watchOS 3.2 simulators

### ISSUE THREAD

**Issue by ikesyo** [travis-ci/travis-ci#7580]
Coming from https://github.com/travis-ci/travis-ci/issues/7532#issuecomment-292067698. It's unfortunate that you now have duplicate watchOS 3.2 simulators instead of iOS 10.2 simulators 😲 

```
-- watchOS 3.2 --
    Apple Watch - 38mm (38C891AB-EC79-43E4-9311-6FDEB12DCF11) (Shutdown)
    Apple Watch - 38mm (35AB3F5D-069B-4B51-B04B-F6AD78C3041E) (Shutdown)
    Apple Watch - 42mm (BE78B202-BB77-4DC8-A6F0-0F648DA89B4A) (Shutdown)
    Apple Watch - 42mm (644B67CA-F6BA-4DE5-A826-CA96D88E3C0C) (Shutdown)
    Apple Watch Series 2 - 38mm (9DF64E1A-A613-4523-A9B7-B98376154190) (Shutdown)
    Apple Watch Series 2 - 38mm (3A12376B-EDE6-48DC-B8E0-CDA704E9D423) (Shutdown)
    Apple Watch Series 2 - 42mm (25EF4763-1942-4A3B-A80F-FC080592B82C) (Shutdown)
    Apple Watch Series 2 - 42mm (D5912402-77B7-44B8-93FD-B00406F46415) (Shutdown)
```

**Comment by emaloney** [comment]
As [noted here](https://github.com/travis-ci/travis-ci/issues/7031#issuecomment-292083796), we are hitting this problem with duplicate watchOS simulators on the Xcode 8.3 image as well. (As an example, we have [a build failing here](https://travis-ci.org/emaloney/MBToolbox/builds/219161787).)

In the past, we've worked around problems like this by switching to another device/OS combo that isn't duplicated. However, that's not possible in this case because _all_ watchOS simulators are duplicated.

@russbishop had [a suggestion](https://github.com/travis-ci/travis-ci/issues/7031#issuecomment-292048332) that worked for us: namely, deleting all the simulator device files and killing the simulator. On re-launch, Xcode will re-create all the default simulators.

To implement Russ's fix, we added the following lines to the `install` section of our `.travis.yml` file:

```
- rm -rf ~/Library/Developer/CoreSimulator
- sudo killall -9 com.apple.CoreSimulator.CoreSimulatorService
```

Because this greatly increases the first launch time of whatever simulator(s) you end up using, we also had to add a `-destination-timeout 300` to our `xcodebuild` commands to make sure our unit tests didn't fail to launch.
